### ISSUE TITLE
Per-channel enable/disable

### ISSUE THREAD

**Issue by Idiosyncratinom** [SkipTo/SkipTo#5]
Some users may nearly always watch a certain Youtuber's videos in their entirety, whilst wishing only to see key moments of others (e.g. ones they're not subscribed to).

**Comment by SkipTo** [comment]
good idea! This can definitely be done with [chrome.storage](https://developer.chrome.com/extensions/storage). If more interest is given to SkipTo, I'll be able to put some time into implementing this.

**Comment by Idiosyncratinom** [comment]
Good job of leaving the link - just had a quick read. Sounds good!
